src/js/utils/template-processor.js

```diff
@@ -149,7 +149,11 @@ export const processor = {
         }
       } else {
         if (part instanceof AttrPart) {
-          part.booleanValue = false;
+          try {
+            part.booleanValue = false;
+          } catch {
+            // attribute can't be turned off, consists of multiple parts.
```

Comment:
question(blocking): I'm assuming this was added/discovered as an edge case during implementation. If yes, can you describe the case where it happened? Should this just be a "silent catch" or should there be some visibility somehow if/when this case occurs?

Reply:
the issue was that the booleanValue throws when setting an attribute consisting of multiple attrparts

https://github.com/muxinc/media-chrome/blob/289611899aeb07c460a6c7261cf128f62fad7ffd/src/js/utils/template-parts.js#L288-L289

I changed it to use the regular `.value` which doesn't need this try / catch. 
I think this is better, silently failing might be unexpected.